src/snowflake/snowpark/modin/plugin/_internal/pivot_utils.py

```diff
@@ -194,6 +194,20 @@ def pivot_helper(
     if ordered_dataframe.queries.get("post_actions"):
         ordered_dataframe = cache_result(ordered_dataframe)
 
+    if pivot_aggr_groupings is None:
+        # When pivot_aggr_groupings is None, there are no `values` to compute on. In that case, we simply return
+        # a DataFrame with no columns, whose index is the result of grouping by the index columns.
+        ordered_dataframe = ordered_dataframe.group_by(
```

Comment:
you can probably just call get_distinct_rows in groupby_utils.py here, it does pretty much the same thing with a dropping of extra columns, just to unify the same usage here.

Reply:
`get_distinct_rows` will also group by the pivot columns though, right? I looked at the method, and it seems that it groups by the projected columns, but here we only want to group by the groupby_snowflake_quoted_identifiers.

Reply:
oops, sorry, i meant get_groups_for_ordered_dataframe in the groupby_utils.py